__fixtures__/test-project/api/src/lib/auth.ts

```diff
@@ -53,24 +53,34 @@ export const hasRole = ({ roles }: { roles: AllowedRoles }): boolean => {
     return false
   }
 
-  if (roles) {
-    if (Array.isArray(roles)) {
-      if (Array.isArray(context.currentUser.roles)) {
-        return context.currentUser.roles?.some((r) => roles.includes(r))
-      } else {
-        roles.some((r) => context.currentUser.roles?.includes(r))
-      }
-    }
+  const currentUserRoles = context.currentUser?.roles as string | string[]
```

Comment:
Why is the typecasting here needed?

Reply:
See here: https://github.com/redwoodjs/redwood/pull/4678/files#r822434312

Reply:
Ohh, sorry, I didn't know how all this worked.
So tasks/test-project/tasks.js is used to generate this fixture, right? That's why this line changed?

Reply:
Yup that's right. You use the `--rebuild-fixture` flag on test project, and it'll update the fixture with the latest template changes, codemods, etc!